exec/java-exec/pom.xml

```diff
@@ -502,6 +502,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-yarn-api</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Is this needed here? Drill-on-YARN uses the YARN API. We did try hard to avoid adding a YARN dependency to Drill itself, since Drill can run stand-alone, under YARN, under K8s, etc.

Reply:
Unfortunately the mini dfs cluster tests failed without this dependency, which is why this a test dependency and not a compile/runtime one luckily...

Reply:
Thanks for the explanation. Not much we can do, I suppose.